deploy-lvmo/deploy.sh

```diff
@@ -55,9 +55,12 @@ if ! ./verify.sh; then
 
     index=0
     for edgecluster in ${ALLEDGECLUSTERS}; do
+        # wipe disks on nodes 
+        wipe_edge_disks
```

Comment:
Wondering (probably unrelated to this PR): Do we need to also clean VGs? PVs? any lvm resource?

Probably not since we are wiping the discs anyways. Should we do it (for good measure) before wiping the disks?

Reply:
My underdatnding is that when you remove the partitioning of each disks, all the informartion regarding the PV is erased, and as the VG metada is not present in the installation because the VG was created in a previous installation the OS only will see empty disks that can be used to create the new PV, LV and VG.